Jenkinsfile

```diff
@@ -30,7 +30,7 @@ properties(
     ]
 )
 final def oses = ['linux':'ubuntu && maven', 'windows':'windows-he']
-final def mavens = env.BRANCH_NAME == 'master' ? ['3.6.x', '3.2.x'] : ['3.2.x']
+final def mavens = env.BRANCH_NAME == 'master' ? ['3.2.x', '3.8.x'] : ['3.8.x']
```

Comment:
the first version of maven will be used to deploy artifacts - maybe better is use the latest version to deploy

Reply:
good point